workspaces/default/themes/base/assets/js/kong/kong.utils.js

```diff
@@ -64,3 +64,196 @@ window.Kong.Utils.getErrorMessage = function (response) {
     return fallbackErrorMessage;
   }
 };
+
+/**
+ * Get value of specified descendent property of an object or object array.
+ *
+ * The `key` argument accepts dot characters (`.`) to deeply traverse input objects
+ * and asterisk characters (`*`) to select all properties of currently selected object.
+ * You can use the asterisk character at any point of the object tree.
+ *
+ * Example:
+ *
+ *   For given input object:
+ *   ```
+ *   {
+ *     info: {
+ *       title: "Hello, World!",
+ *     },
+ *     tags: {
+ *       first: {
+ *         name: "First tag",
+ *       },
+ *       second: {
+ *         name: "Second tag",
+ *       },
+ *     },
+ *   }
+ *   ```
+ *   `info.title` key will get the value of the `title` property in the `info` object, and
+ *
+ *   `tags.*.name` key will get the values of the `name` property in all object properties inside the `tags` object.
+ *
+ * @param {Object|Object[]} objOrObjs - An object or array of objects to extract the values from
+ * @param {string} key - Key to get the properties from given object(s)
+ * @returns {string[]} Values of requested properties
+ */
+window.Kong.Utils.getDescendantProperties = function (objOrObjs, key) {
```

Comment:
```suggestion
  window.Kong.Utils.getDescendantProperties = function (data, key) {
```
`objOrObjs` seems bloated name

Same remark for `Descendant` to `Nested`